nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/config/ESClientInitializationDescriptor.java

```diff
@@ -42,6 +42,12 @@
     @XNode("password")
     protected String password;
 
+    @XNode("sslKeystorePath")
+    protected String sslKeystorePath;
+
+    @XNode("sslKeystorePassword")
+    protected String sslKeystorePassword;
+
```

Comment:
a detail, but it's a good practice to keep since annotations on xmap-annotated attributes too